taxcalc/calculate.py

```diff
@@ -322,6 +326,8 @@ def mtr(self, variable_str='e00200p',
             self.records.e00900 = seincome_var + finite_diff
         elif variable_str == 'e00650':
             self.records.e00600 = divincome_var + finite_diff
+        elif variable_str == 'e26270':
+            self.records.e02000 = schEincome_var + finite_diff
```

Comment:
finite_diff is already being added to e26270 in the line:
setattr(self.records, variable_str, variable + finite_diff)

However, e26270 is included in e02000, so you have to increase e02000 as well. If not, then you would be calculating the MTR from adding finite_diff to e26270 but substracting it from rental income.